File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/ICategoryFragment.java
package com.artmurka.artmurkaapp.android.views.fragments.interfaces;

import com.artmurka.artmurkaapp.model.pojo.itemlist.categories.Success;

import java.util.List;


public interface ICategoryFragment {

    void showCategories(List<Success> categoriesList); //show categories
    void setToolBarName(String name);
    void showError(String error); //show error, if error
}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/ICategoryFragment.kt
package com.artmurka.artmurkaapp.android.views.fragments.interfaces

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.categories.Success
import com.artmurka.artmurkaapp.presenter.PresenterView


interface ICategoryFragment : PresenterView {

    fun showCategories(categoriesList: List<Success>)  //show categories
    fun setToolBarName(name: String)
    fun showError(error: String)  //show error, if error
}
